ScanToShop/Managers/DatabaseManager.m

```diff
@@ -7,16 +7,50 @@
 //
 
 #import "DatabaseManager.h"
+#import "AlertManager.h"
 
 @implementation DatabaseManager
 
-+ (PFFileObject *)getPFFileFromImage: (UIImage * _Nullable)image {
- 
-    if (!image) {
-        return nil;
-    }
++(BOOL) registerUserInParse:(UIViewController *)vc User:(User *)user {
```

Comment:
Having Parse in the name leaks implementation details. If you change to Realm you don't want to have to rename this. Better to just have `saveUser` or something.